File: src/codes/Users_Manual/Tutorial/multiple/geometry_1.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Utils;
~var link = new Nop();
Controller.Open([
   new AUTD3(pos: new Point3(-AUTD3.DeviceWidth, 0, 0), rot: Quaternion.Identity),
   new AUTD3(pos: Point3.Origin, rot: Quaternion.Identity)
], link)
~;

File: src/codes/Users_Manual/Tutorial/multiple/geometry_1.rs
# use autd3::prelude::*;
# 
# fn main() -> Result<(), Box<dyn std::error::Error>> {
# let link = autd3::link::Nop::new();
# let _ =
Controller::open(
    [
        AUTD3 {
            pos: Point3::new(-AUTD3::DEVICE_WIDTH, 0., 0.),
            rot: UnitQuaternion::identity(),
        },
        AUTD3 {
            pos: Point3::origin(),
            rot: UnitQuaternion::identity(),
        },
    ],
    link,
)?;
#     Ok(())
# }